commands/channel_test.go

```diff
@@ -2176,3 +2176,169 @@ func (s *MmctlUnitTestSuite) TestRenameChannelCmd() {
 		s.Require().Equal(printer.GetLines()[0], updatedChannel)
 	})
 }
+
+func (s *MmctlUnitTestSuite) TestRemoveChannelCmd() {
+	userArg := "user@example.com"
```

Comment:
This should be userEmail to be more clear

Reply:
Agree, again, I've named this variable like that because I've seen it already in other part of the code so to keep consistency, but I'll change it.